### PR TITLE
update: new get started with kmm

### PR DESCRIPTION
**Pull request by danil-pavlov**

### CODE REVIEW

File: docs/topics/kmm/kmm-getting-started.md [hunk 1]
```diff
@@ -1,22 +1,54 @@
...
+
+[Kotlin Multiplatform](multiplatform.md) technology also supports other platforms, including JavaScript, Linux, WebAssembly,
+and [more](mpp-dsl-reference.md#targets).
+
+See [supported platforms for Kotlin multiplatform programming](mpp-supported-platforms.md).
+
+## Start from scratch
+
+* [Set up the environment for cross-platform mobile development](kmm-setup.md)
+* [Create your first multiplatform mobile app using the IDE wizard](kmm-create-first-app.md)
```
Comment (katia-energizer) [inline docs/topics/kmm/kmm-getting-started.md:39]: We haven't agreed on capitalization of `multiplatform`. I would avoid using it and replace with synonyms such cross-platform mobile app and app that works on Android and iOS.

File: docs/topics/kmm/kmm-getting-started.md [hunk 2]
```diff
@@ -1,22 +1,54 @@
-[//]: # (title: Getting started)
+[//]: # (title: Get started with Kotlin Multiplatform Mobile)
+[//]: # (description: cross platform mobile development, android vs ios development)
```
Comment (katia-energizer) [inline docs/topics/kmm/kmm-getting-started.md:2]: `description` should be a summary of 155-160 characters that motivates people to click this search result. Please let me know if you need help here.

File: docs/topics/kmm/kmm-getting-started.md [hunk 3]
```diff
@@ -1,22 +1,54 @@
...
+> Beta is expected in spring 2022. See [KMM Beta Roadmap](https://youtu.be/LE-Dy9XCLxk) video
+> to learn about upcoming features.
+>
+{type="note"}
+
+Kotlin Multiplatform Mobile (KMM) is an SDK designed to simplify development of cross-platform mobile applications.
+You can share common code between iOS and Android apps and write platform-specific code only where it's necessary.
+For example, to implement a native UI or when working with platform-specific APIs.
 
 Watch the introductory [video](https://www.youtube.com/watch?v=mdN6P6RI__k), in which Kotlin Developer Advocate Ekaterina
```
Comment (katia-energizer) [inline docs/topics/kmm/kmm-getting-started.md:16]: Kate is not a developer advocate anymore. Please contact her to agree on her title.

File: docs/topics/kmm/kmm-getting-started.md [hunk 4]
```diff
@@ -1,22 +1,54 @@
...
-you'll set up an environment and prepare for creating your first cross-platform mobile application with KMM.
+Petrova explains what Kotlin Multiplatform Mobile is and how you can use it in your project. With Ekaterina,
+you'll set up an environment and prepare to create your first cross-platform mobile application:
 
 <video href="mdN6P6RI__k" title="Kotlin Multiplatform Multiverse, Episode 1: Meet KMM!"/>
 
-## Start KMM from scratch
+You can also check out other videos about [Kotlin Multiplatform Multiverse](https://www.youtube.com/playlist?list=PLlFc5cFwUnmy_oVc9YQzjasSNoAk4hk_C) on YouTube.
+
+## Supported platforms
```
Comment (katia-energizer) [inline docs/topics/kmm/kmm-getting-started.md:24]: I thought that we agreed to move supported platforms to our general file with platforms and include a link here. I am not sure that such detailed description is required for Get started. We could make it high level and move all details to the platform reference.

File: docs/topics/kmm/kmm-getting-started.md [hunk 5]
```diff
@@ -1,22 +1,54 @@
...
+and [more](mpp-dsl-reference.md#targets).
+
+See [supported platforms for Kotlin multiplatform programming](mpp-supported-platforms.md).
+
+## Start from scratch
+
+* [Set up the environment for cross-platform mobile development](kmm-setup.md)
+* [Create your first multiplatform mobile app using the IDE wizard](kmm-create-first-app.md)
+* [Check out the list of sample projects](kmm-samples.md)
+* [Introduce Kotlin for cross-platform mobile app development to your team](kmm-introduce-your-team.md)
```
Comment (katia-energizer) [inline docs/topics/kmm/kmm-getting-started.md:41]: `Introduce Kotlin for cross-platform mobile app development to your team` seems too long. I'd try to shorten it. We can at least `mobile`.

File: docs/topics/kmm/kmm-getting-started.md [hunk 6]
```diff
@@ -1,22 +1,54 @@
...
+
+## Make your Android application work on iOS
 
-* [Set up your environment for KMM development](kmm-setup.md).
-* [Create your first KMM application](kmm-create-first-app.md) using the IDE wizard.
-* [Check the KMM sample projects](kmm-samples.md) for inspiration.
+If you already have an Android mobile application and want to make it cross-platform:
```
Comment (katia-energizer) [inline docs/topics/kmm/kmm-getting-started.md:45]: Please check `cross-platform` with copyeditors in this case.

File: docs/topics/kmm/kmm-getting-started.md [hunk 7]
```diff
@@ -1,22 +1,54 @@
...
+* [Make a sample Android application work well on iOS](kmm-integrate-in-existing-app.md)
 
-If you already have a mobile application and want to make it cross-platform:
+## Get help
 
-* [Set up your environment for KMM development](kmm-setup.md).
-* [Make a sample Android application work well on iOS](kmm-integrate-in-existing-app.md).
+* **Kotlin Slack**: Get an [invite](https://surveys.jetbrains.com/s3/kotlin-slack-sign-up) and join the [#multiplatform](https://kotlinlang.slack.com/archives/C3PQML5NU) channel
+* **StackOverflow**: Subscribe to the [“kotlin-multiplatform” tag](https://stackoverflow.com/questions/tagged/kotlin-multiplatform)
+* **Kotlin issue tracker**: [Report a new issue](https://youtrack.jetbrains.com/newIssue?project=KT)
```
Comment (katia-energizer) [inline docs/topics/kmm/kmm-getting-started.md:54]: Please check with Sergei Pecherkin whether we wan to add any tags/fields to the URL. The current URL just creates a Kotlin ticket.

**Review by katia-energizer — CHANGES_REQUESTED** [review]
Please take a look at comments and provide a link to staging.

File: docs/topics/kmm/kmm-getting-started.md (hunk 4 above)
Comment (danil-pavlov) [reply docs/topics/kmm/kmm-getting-started.md:24]: Trimmed platform descriptions following Kate's suggestions

File: docs/topics/kmm/kmm-getting-started.md (hunk 7 above)
Comment (danil-pavlov) [reply docs/topics/kmm/kmm-getting-started.md:55]: Sergei advised to leave the link as is

File: docs/topics/kmm/kmm-getting-started.md (hunk 6 above)
Comment (danil-pavlov) [reply docs/topics/kmm/kmm-getting-started.md]: Checked. Decided to leave as is for consistency

File: docs/topics/kmm/kmm-getting-started.md [hunk 8]
```diff
@@ -1,22 +1,55 @@
-[//]: # (title: Getting started)
+[//]: # (title: Get started with Kotlin Multiplatform Mobile)
+[//]: # (description: Simplify cross-platform app development with Kotlin Multiplatform Mobile. Create a single codebase
+for the business logic of your iOS and Android apps.)
 
-## Get familiar with KMM
+> Kotlin Multiplatform Mobile is in [Alpha](components-stability.md). Language features and tooling may change in future
+> Kotlin versions.
+> 
+> Beta is expected in spring 2022. See [KMM Beta Roadmap](https://youtu.be/LE-Dy9XCLxk) to learn about upcoming features.
```
Comment (katia-energizer) [inline docs/topics/kmm/kmm-getting-started.md:8]: We have renamed the video. I think that it's better to link to the [corresponding blog post](https://blog.jetbrains.com/kotlin/2021/10/kmm-beta-roadmap-video-highlights/) (will be renamed as well) that describes highlights and embeds this video. Please consult with @KaterinaPetrova .

File: docs/topics/kmm/kmm-getting-started.md [hunk 9]
```diff
@@ -1,22 +1,55 @@
...
+
+* Android applications and libraries
+* [Android NDK](https://developer.android.com/ndk) (ARM64 and ARM32)
+* Apple iOS devices (ARM64 and ARM32) and simulators
+* Apple watchOS devices (ARM64 and ARM32) and simulators
+
+[Kotlin Multiplatform](multiplatform.md) technology also supports other platforms, including JavaScript, Linux, WebAssembly,
+and [more](mpp-dsl-reference.md#targets).
+
+See the [full list of supported platforms](mpp-supported-platforms.md) for Kotlin Multiplatform programming.
```
Comment (katia-energizer) [inline docs/topics/kmm/kmm-getting-started.md:34]: I would combine the last two paragraphs or maybe even include the link to platform descriptions in the first paragraph.
It'd help keep shorter this section with additional information.

File: docs/topics/kmm/kmm-getting-started.md (hunk 7 above)
Comment (katia-energizer) [reply docs/topics/kmm/kmm-getting-started.md:55]: OK

**Review by katia-energizer — COMMENTED** [review]
Thank you! Minor comments. Please check

File: docs/topics/kmm/kmm-getting-started.md (hunk 9 above)
Comment (danil-pavlov) [reply docs/topics/kmm/kmm-getting-started.md]: True, targets in MPP DSL reference and Supported platforms seem to have similar information. Left one link only and combined into one paragraph

**Pull request merged**